functional-tests/utils.py

```diff
@@ -225,12 +225,12 @@ def submit_checkpoint(idx: int, seqrpc, manual_gen: ManualGenBlocksConfig | None
 
 def check_submit_proof_fails_for_nonexistent_batch(seqrpc, nonexistent_batch: int):
     """
-    This check requires that subnitting nonexistent batch proof fails
+    This check requires that submitting nonexistent batch proof fails
```

Comment:
```suggestion
    Requires that submitting nonexistent batch proof fails
```
"This" and "check" are redundant. The function name is `check_foo`